src/View/Metadata.php

```diff
@@ -36,6 +38,10 @@ public function render(string $type): string
             return '<title>' . $this->title . '</title>';
         }
 
+        if ($type === 'locale') {
+            return $this->locale;
```

Comment:
I am a little concerned by this use, honestly. All of the rest of this class is intended managing metadata and script/style **tags**. Just returning the raw locale value makes it a very odd and feels like it's just making a new bucket to store random information in. 

I just looked it up and there is no standard tag to specify locale, with the exception of adding `lang="x"` on the `html` tag. If this function were to return the full `lang="x"` string intended for the html tag, I think it would make sense. 

As it is, this doesn't save anything over `service('request')->getLocale()`and I don't feel is needed here.

Reply:
Yes, I get your point, it is overcomplicated. Could we just use the request service getLocale method in the template directly:
`<html lang="<?= service('request')->getLocale() ?>">` , thus avoiding the MetaData lib entirely?

I am submitting patches to Bonfire for two reasons: 

- one is that I do intend to use this software (thus far I was using self-assembled admin panel with ionAuth for my projects, that I am not satisfied with; Bonfire seems to have much better mechanisms for dealing with menus, widgets and modularising code). 
- The other is, that I am trying to learn OOP and I hope to learn from doing things for Bonfire. 

Therefore I do not expect my submissions to always make sense or be accepted.